staging/src/k8s.io/kubectl/pkg/cmd/apply/patcher.go

```diff
@@ -147,24 +147,24 @@ func (p *Patcher) patchSimple(obj runtime.Object, modified []byte, namespace, na
 		if patch == nil {
 			patch, err = p.buildStrategicMergeFromBuiltins(p.Mapping.GroupVersionKind, original, modified, current)
 			if err != nil {
-				return nil, nil, errors.Wrapf(err, createPatchErrFormat, original, modified, current)
+				return false, nil, errors.Wrapf(err, createPatchErrFormat, original, modified, current)
 			}
 		}
 	}
 
 	if string(patch) == "{}" {
-		return patch, obj, nil
+		return false, obj, nil
 	}
 
 	if p.ResourceVersion != nil {
 		patch, err = addResourceVersion(patch, *p.ResourceVersion)
 		if err != nil {
-			return nil, nil, errors.Wrap(err, "Failed to insert resourceVersion in patch")
+			return true, nil, errors.Wrap(err, "Failed to insert resourceVersion in patch")
```

Comment:
Is this return `true` by intention? I would have expected `false` especially since the request has not been sent by this point.